docs/examples/transforms/TransformsEval.ipynb

```diff
@@ -0,0 +1,1622 @@
+{
```

Comment:
<p>could we clean out those </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/9005/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>